js/index.js

```diff
@@ -1,7 +1,84 @@
 // Iteration 1: Names and Input
+const hacker1 = "Eliher";
+console.log("The driver's name is " + hacker1);
 
+const hacker2 = "Luis";
+console.log("The navigator's name is " + hacker2);
 
 // Iteration 2: Conditionals
+if(hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+  }else if(hacker2.length > hacker1.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+  }else{
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
 
+// Iteration 3: Loops
+let driver = hacker1;
+let name = "";
+for(let i = 0; i < driver.length; i++){
+  name = name + driver[i] + " ";
+}
+console.log(name.toUpperCase());
 
-// Iteration 3: Loops
\ No newline at end of file
+//
+let navigator = hacker2;
+let reversedName = "";
+for(let i = 0; i < navigator.length; i++){
+  reversedName = navigator[i] + reversedName;
+}
+console.log(reversedName);
+
+// Done with a Function 
+/* function reverseString(str) {
+    return str.split("").reverse().join("");
+}
+console.log(reverseString(hacker2));
+*/
+
+let n = hacker1.localeCompare(hacker2);
+
+switch(n){
```

Comment:
Buena manera de solucionarlo, 
la mejor practica seria una funcion, les pongo un ejemplo de otra solucion de esta manera.

`if(hacker1.length > hacker2.length){`
`console.log(`The Driver has the longest name, it has ${hacker1.length} characters`)`
`} else if (hacker1.length < hacker2.length) {`
`console.log(`The Nav has the longest name, it has ${hacker2.length} characters`)`
`} else {`
`console.log("same")`
`}`